aws/resource_aws_lb.go

```diff
@@ -111,6 +111,11 @@ func resourceAwsLb() *schema.Resource {
 							Required: true,
 							ForceNew: true,
 						},
+						"outpost_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+							Optional: true,
```

Comment:
Since this doesn't appear to be configurable, it should not be declared with `Optional` 👍 

```suggestion
```